docs/bootstrap-governance/governance.md

```diff
@@ -0,0 +1,137 @@
+# Governance of Executable Books
+
+```{note}
+_All contributors_ to the Executable Books Project (including but not limited to individuals contributing code, providing technical support, teaching workshops, and engaging in discussions about changes to EBP policy or the MyST Specification) adhere to the [Code of Conduct](policy:coc).
```

Comment:
_adhere to_ -> _are expected to adhere to_ or _implicitly agree to abide by_

I think that's an important distinction as the former is a description of idealized behavior of the community, while the latter sets out our expectations for community members. (And apologies if I'm requesting a change to text I originally drafted! I'm reading on this subject now.)